src/main/java/codeu/helper/BotActions.java

```diff
@@ -0,0 +1,130 @@
+package codeu.helper;
+
+import java.io.IOException;
+import java.time.Instant;
+import java.util.List;
+import java.util.UUID;
+
+import javax.servlet.http.HttpServletResponse;
+
+import codeu.model.data.Conversation;
+import codeu.model.data.Message;
+import codeu.model.data.User;
+import codeu.model.store.basic.ActivityStore;
+import codeu.model.store.basic.ConversationStore;
+import codeu.model.store.basic.MessageStore;
+import codeu.model.store.basic.UserStore;
+
+import com.google.common.collect.Lists;
+
+/** Helper class responsible of the bot's actions */
+public class BotActions {
+
+    private ActivityStore activityStore;
+    private User user;
+    private ConversationStore conversationStore;
+    private HttpServletResponse response;
+    private MessageStore messageStore;
+    private UserStore userStore;
+
+    public BotActions(String username) {
```

Comment:
It might be easier to just use the store's respective getInstance() methods.

Reply:
Right. Thought I did it, thanks.